spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/EurekaClientAutoConfiguration.java

```diff
@@ -212,18 +203,14 @@ public ApplicationInfoManager eurekaApplicationInfoManager(
 		@Autowired
 		private ApplicationContext context;
 
-		@Autowired(required = false)
-		private DiscoveryClientOptionalArgs optionalArgs;
```

Comment:
I was affraid of that one. The only workaound I can think of is providing an abstraction of `DiscoveryClientOptionalArgs` which will be compatible with injection without forcing the dependency on Jersey. I'll rework that bit.